root/etc/s6-overlay/s6-rc.d/svc-backend/run

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/with-contenv bash
+# shellcheck shell=bash
+
+HOME=/data exec \
+    s6-notifyoncheck -d -n 300 -w 1000 -c "nc -z localhost 3000" \
+        cd /emulatorjs s6-setuidgid abc node index.js
```

Comment:
```suggestion
cd /emulatorjs  || exit 1
HOME=/data exec \
    s6-notifyoncheck -d -n 300 -w 1000 -c "nc -z localhost 3000" \
        s6-setuidgid abc node index.js
```

Just as a good practice, `cd whatever || exit 1`
Also, I wasn't sure if that should (or had to) be done in the exec or if it would be fine above. I'm also curious if
```shell
HOME=/data
```
could be
```shell
HOME=/data
export HOME
```
and thus not inline with the `exec` but that's more out of curiosity than necessity or preference. My reason for wondering is whether exported vars carry into the env when using `s6-setuidgid abc` which I would imagine they do.